src/data/actions/courseInfo.test.js

```diff
@@ -224,6 +230,8 @@ describe('courseInfo edit course actions', () => {
     expect(changeMock).toHaveBeenNthCalledWith(8, 'in_year_value.per_lead_international', 20);
     expect(changeMock).toHaveBeenNthCalledWith(9, 'in_year_value.per_click_usa', 25);
     expect(changeMock).toHaveBeenNthCalledWith(10, 'in_year_value.per_click_international', 30);
+    expect(changeMock).toHaveBeenNthCalledWith(10, 'additional_metadata.certificate_info.heading', 'Test Certificate');
+    expect(changeMock).toHaveBeenNthCalledWith(10, 'additional_metadata.certificate_info.blurb', 'test blurb');
```

Comment:
```suggestion
expect(changeMock).toHaveBeenNthCalledWith(12, 'additional_metadata.certificate_info.heading', 'Test Certificate');
```
update the Nth Called value for the rest of them